pom.xml

```diff
@@ -1020,7 +1020,6 @@
 						<failOnError>true</failOnError>
 						<doclint>none</doclint>
 						<additionalJOption>${javadoc.additionalJOption}</additionalJOption>
-						<sourcepath>${project.build.sourceDirectory}</sourcepath>
```

Comment:
I should remove this from the jpms branch too!

Reply:
Maybe `<source>8</source>` can be safely removed for jpms branch :)

Reply:
This is already done: https://github.com/apache/wicket/blob/d0c80aeb774414fcdf11129a98c930ce4f79f25c/pom.xml#L1017